src/utils/get-article-share-links.js

```diff
@@ -0,0 +1,8 @@
+export const getArticleShareLinks = (title, url) => {
+    const facebookUrl = `https://www.facebook.com/sharer/sharer.php?u=${url}`;
+    const twitterUrl = `https://twitter.com/intent/tweet?url=${url}&text=${encodeURIComponent(
+        `Here is a post from @mongodb on "${title}"`
```

Comment:
Let's go with just `title`. They're adding a new 'title' field to the twitter meta as well, but I don't think we have access to that data here.

Reply:
So the text would be just `${title}` and nothing more?